pkg/apis/policies/v1/policyserver_webhook.go

```diff
@@ -29,17 +34,25 @@ import (
 // log is for logging in this package.
 var policyserverlog = logf.Log.WithName("policyserver-resource")
 
-func (ps *PolicyServer) SetupWebhookWithManager(mgr ctrl.Manager) error {
-	err := ctrl.NewWebhookManagedBy(mgr).
-		For(ps).
-		Complete()
-	if err != nil {
-		return fmt.Errorf("failed enrolling webhook with manager: %w", err)
+// SetupPolicyServerWebhookWithManager returns a function that can be used to setup the PolicyServer webhook with the manager.
+// This is needed since we are using github.com/ereslibre/kube-webhook-wrapper and will be removed once we no longer use it.
+func SetupPolicyServerWebhookWithManager(deploymentsNamespace string) func(mgr ctrl.Manager) error {
+	return func(mgr ctrl.Manager) error {
+		ps := &PolicyServer{}
+
+		err := ctrl.NewWebhookManagedBy(mgr).
+			For(ps).
+			WithValidator(&policyServerValidator{k8sClient: mgr.GetClient(), deploymentsNamespace: deploymentsNamespace}).
+			Complete()
+		if err != nil {
+			return fmt.Errorf("failed enrolling webhook with manager: %w", err)
+		}
+
+		return nil
 	}
-	return nil
 }
 
-//+kubebuilder:webhook:path=/mutate-policies-kubewarden-io-v1-policyserver,mutating=true,failurePolicy=fail,sideEffects=None,groups=policies.kubewarden.io,resources=policyservers,verbs=create;update,versions=v1,name=mpolicyserver.kb.io,admissionReviewVersions={v1,v1beta1}
+// +kubebuilder:webhook:path=/mutate-policies-kubewarden-io-v1-policyserver,mutating=true,failurePolicy=fail,sideEffects=None,groups=policies.kubewarden.io,resources=policyservers,verbs=create;update,versions=v1,name=mpolicyserver.kb.io,admissionReviewVersions={v1,v1beta1}
```

Comment:
This was changed by the code formatted, I hope that doesn't break the code generator.
I think this didn't happen, I guess the change to the `manifest.yaml` file inside of this PR has been done by the code generator. Am I right?

Reply:
it works either way. We are not very consistent either we use the version with the space in other part of the code alongside the one with the one with no space.

Reply:
Fine with me, I don't have a strong opinion about this formatting. I just want the code generator to work :)